content/rdi/installation/install-k8s.md

```diff
@@ -186,16 +217,16 @@ The `create` command will create a BDB named `redis-di-1` in your cluster. You w
 Run `deploy` command to deploy the configuration in the ConfigMap to the remote redis-di instance:
 
 ```bash
-kubectl exec -it pod/redis-di-cli -- redis-di deploy
+kubectl exec -it pod/redis-di-cli-<id> -- redis-di deploy
 ```
 
 > Read more about deploying data transformation jobs when the RDI CLI is deployed as a Kubernetes pod [here](../data-transformation-pipeline.md#deploy-configuration).
 
 ### Validate the installation
 
-Run `kubectl exec -it pod/redis-di-cli -- redis-di status` to check the status of the installation.
+Run `kubectl exec -it pod/redis-di-cli-<id> -- redis-di status` to check the status of the installation.
 
-> Note that it is OK to see the warning of "No streams found" since we have not yet set up a Debezium source connector. We will do this in the next step.
+> Note: It is OK to see the warning of "No streams found" since we have not yet set up a Debezium source connector. We will do this in the next step.
```

Comment:
```suggestion
> Note: It is okay to see the warning "No streams found" since the Debezium source connector has not yet been set up. This will be done in the next step.
```